next.config.js

```diff
@@ -1,3 +1,9 @@
 const withOffline = require('next-offline')
 
-module.exports = withOffline({ dontAutoRegisterSw: true })
+module.exports = withOffline({
+  target: 'serverless',
+  dontAutoRegisterSw: true,
+  experimental: {
```

Comment:
```suggestion
  workboxOpts: {
    swDest: path.join(__dirname, 'public/service-worker.js'),
  },
  experimental: {
```

Reply:
@ijjk this it now working, but the exported `service-worker.js` looks incorrect. Were there breaking changes with how the files are exported? 

(https://carbon-9gvevwmgy.now.sh/service-worker.js)
```js
self.__precacheManifest = [
  {
    "url": "/_next/static/A6_zdG4b__6z6u6~Yvxr6/pages/_app.js",
    "revision": "baaf6a34d4e46f39ab1b"
  },
  {
    "url": "/_next/static/A6_zdG4b__6z6u6~Yvxr6/pages/_error.js",
    "revision": "82986cfd500b031b842d"
  },
  {
    "url": "/_next/static/A6_zdG4b__6z6u6~Yvxr6/pages/about.js",
    "revision": "7a52c3d6947ad4d6069a"
  },
  {
    "url": "/_next/static/A6_zdG4b__6z6u6~Yvxr6/pages/embed.js",
    "revision": "0a1aae8cef7c59539916"
  },
  {
    "url": "/_next/static/A6_zdG4b__6z6u6~Yvxr6/pages/index.js",
    "revision": "f74a6f915575adf6d224"
  },
  {
    "url": "/_next/static/chunks/4f3d.4f29a31c9b2405e7f1eb.js",
    "revision": "f2ec1d304f122dd7f441"
  },
  {
    "url": "/_next/static/chunks/7021.c9608c316ac8728ec42c.js",
    "revision": "24e31e618e3ec49d740e"
  },
  {
    "url": "/_next/static/chunks/8fda.208c4594c728152ed538.js",
    "revision": "883598af10579d079b50"
  },
  {
    "url": "/_next/static/chunks/cbfc.8ec4593a7acccfa77c95.js",
    "revision": "9edebaef2c40e9ff8f60"
  },
  {
    "url": "/_next/static/chunks/commons.c116e5e03b31e084745e.js",
    "revision": "3ca1ce41bfb359c73baf"
  },
  {
    "url": "/_next/static/chunks/ea35.67b67cb20915e22ef28c.js",
    "revision": "dd9b75a7b744b8f1bae9"
  },
  {
    "url": "/_next/static/runtime/main-d1b8e7abd637fc72076d.js",
    "revision": "3577e73338e258137e8c"
  },
  {
    "url": "/_next/static/runtime/webpack-0947b9a85b5e00afae80.js",
    "revision": "a7356b692cf50ee77e1c"
  }
];

/**
 * Welcome to your Workbox-powered service worker!
 *
 * You'll need to register this file in your web app and you should
 * disable HTTP caching for this file too.
 * See https://goo.gl/nhQhGp
 *
 * The rest of the code is auto-generated. Please don't update this file
 * directly; instead, make changes to your Workbox build configuration
 * and re-run your build process.
 * See https://goo.gl/2aRDsh
 */

importScripts("https://storage.googleapis.com/workbox-cdn/releases/4.3.1/workbox-sw.js");

importScripts(
  
);

self.addEventListener('message', (event) => {
  if (event.data && event.data.type === 'SKIP_WAITING') {
    self.skipWaiting();
  }
});

/**
 * The workboxSW.precacheAndRoute() method efficiently caches and responds to
 * requests for URLs in the manifest.
 * See https://goo.gl/S9QRab
 */
self.__precacheManifest = [].concat(self.__precacheManifest || []);
workbox.precaching.precacheAndRoute(self.__precacheManifest, {});
```

Reply:
Why does it look incorrect, are there files not there that used to be?

Reply:
@ijjk it's not that it is incorrect, but rather that the service worker is being registered, but failing when items are requested (see deployment)

Do we need to add this line back? https://github.com/dawnlabs/carbon/pull/782/files#diff-96c813af76efae3c872399365c060e54L9